test/features/steps/const.py

```diff
@@ -62,7 +62,7 @@
                         destructive, especially during the "storage" stage.
                         The /root/.ssh/id_rsa key will be overwritten unless
                         the option "--no-overwrite-sshkey" is used)
-  -t TEMPLATE, --template TEMPLATE
+  -t {ocfs2}, --template {ocfs2}
```

Comment:
Why this change? I think that template was a better option. It's true that by now we only have `ocfs2. What if we add some new option in the future, the name won't have sense anymore